### PR TITLE
i/6636: Prefix font styles with the `.ck-content` class.

### LINKED ISSUE #6636: Font size styles (`.text-*`) do not use `.ck-content` prefix
## 📝 Provide detailed reproduction steps (if any)

As described. They are content styles so they should be prefixed with `.ck-content`. Because of that they are missing in https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/content-styles.html (this page is updated before every release so it's not part of this task).

### PR DESCRIPTION
**Pull request by panr**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Font size styles should be prefixed by the `.ck-content` class. Closes ckeditor/ckeditor5#6636.

### CODE REVIEW

**Review by oleq — CHANGES_REQUESTED** [review]
Changes in the guide are necessary https://github.com/ckeditor/ckeditor5-font/blob/i/6636/docs/features/font.md#L97-L111.

Note: You shouldn't use PostCSS syntax there in docs, these are for integrations that may or may not use it.

**Comment by panr** [comment]
@oleq thanks for pointed it out! Changed. 👍

File: theme/fontsize.css [hunk 1]
```diff
@@ -3,18 +3,24 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
-.text-tiny {
-	font-size: .7em;
-}
+/* The values should be synchronized with the "FONT_SIZE_PRESET_UNITS" object in the "/src/fontsize/utils.js" file. */
```
Comment (pomek) [inline theme/fontsize.css:6]: This line was removed in 2f33bedced933ca908efe351f178c3db42bb49c9 and as I can see, it appeared again :D

Comment (oleq) [reply theme/fontsize.css:6]: If it is unnecessary, please get rid of it on master.

**Pull request merged**